pom.xml

```diff
@@ -1571,6 +1571,8 @@
                         <skip>${skipVerifications}</skip>
                     </configuration>
 
+                    <!-- // @todo[question]: Given that the plugin now uses Groovy 3.0.8, should this be set here?-->
+                    <!-- ref: https://github.com/mulesoft/mule-assembly-verifier-plugin/blob/31e94d9efcf987b935191f3af17633147d7c8e43/pom.xml#L38-L42 -->
```

Comment:
@aiannucci Ale, did I dream it, our we talk about deleting this configuration dependency? If so, are we 100% sure this won't mess up with anything Groovy-related on Mule's side?

Reply:
[see](https://github.com/mulesoft/mule/pull/10550#discussion_r679568606)

Reply:
Sure, we should delete it